scripts/validate_diff_accounts.sh

```
@@ -0,0 +1,59 @@
+
+for file in $changed_files; do
+  if [[ $file =~ src/configurations/destinations/([^/]+)/db-config\.json ]]; then
+    # Main destination config file changed
+    dest="${BASH_REMATCH[1]}"
+    if [[ ! " ${destinations[@]} " =~ " ${dest} " ]]; then
+      destinations+=("$dest")
+    fi
+  elif [[ $file =~ src/configurations/destinations/([^/]+)/accounts/ ]]; then
+    # Account config file changed
+    dest="${BASH_REMATCH[1]}"
+    if [[ ! " ${destinations[@]} " =~ " ${dest} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)